PennMobile/Dining/SwiftUI/Views/Venue/DiningVenueView.swift

```diff
@@ -46,13 +59,20 @@ struct DiningVenueView: View {
             }
         }
         .listStyle(.plain)
+        .sheet(isPresented: $alertAccepted) {
+            DiningLoginNavigationView(showSheetView: self.$alertAccepted)
+        }
+        .alert(isPresented: $alertIsShowing) {
+            showCorrectAlert()
+        }
```

Comment:
Does attaching the `.alert`  modifier on the custom header not work? It seems more intuitive for the alert binding to be placed on the custom header view to avoid the issue of passing down Optional Bool binding and creating a function to determine the type of alert to throw.